### ISSUE TITLE
ModuleNotFoundError: No module named 'raven._compat'

### ISSUE THREAD

**Issue by douglatornell** [npilon/pyramid_crow#9]
https://github.com/getsentry/raven-python/pull/812 which was released in raven-6.2.0 on 21-Sep-2017 causes a `ModuleNotFoundError` in pyramid_crow-0.4.0. Rolling raven back to 6.1.0 eliminates the error.

Traceback:
```
Error in sys.excepthook:
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid_crow/processors.py", line 3, in <module>
    from raven._compat import (
ModuleNotFoundError: No module named 'raven._compat'

Original exception was:
Traceback (most recent call last):
  File "/home/sealinkd/miniconda/envs/sealinkd/bin/pserve", line 11, in <module>
    sys.exit(main())
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid/scripts/pserve.py", line 32, in main
    return command.run()
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid/scripts/pserve.py", line 229, in run
    app = loader.get_wsgi_app(app_name, config_vars)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/plaster_pastedeploy/__init__.py", line 131, in get_wsgi_app
    global_conf=defaults)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/paste/deploy/loadwsgi.py", line 203, in invoke
    app = context.app_context.create()
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/home/sealinkd/SealinkD/sealinkd/__init__.py", line 31, in main
    with Configurator(settings=settings) as config:
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid/config/__init__.py", line 345, in __init__
    exceptionresponse_view=exceptionresponse_view,
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid/config/__init__.py", line 463, in setup_registry
    self.include(inc)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid/config/__init__.py", line 839, in include
    c(configurator)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid_crow/__init__.py", line 138, in includeme
    config.scan(ignore='pyramid_crow.tests')
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid/config/__init__.py", line 1043, in scan
    ignore=ignore)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/venusian/__init__.py", line 230, in scan
    __import__(modname)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.6/site-packages/pyramid_crow/processors.py", line 3, in <module>
    from raven._compat import (
ModuleNotFoundError: No module named 'raven._compat'
```

**Comment by npilon** [comment]
Thank you; I've released 0.4.1 which should fix this.

**Comment by douglatornell** [comment]
Thanks for the quick fix and new release! I just updated an environment to 0.4.2 and raven-6.2.1 and things are working smoothly.
